pages/platform/kubernetes-k8s/backing-up-volumes-using-stash/guide.en-asia.md

```diff
@@ -171,6 +171,13 @@ secret/s3-secret created
 
 The easiest way to install Stash is via Helm, using the chart from [AppsCode Charts Repository](https://github.com/appscode/charts).
 
+First of all, you need to [get a license](https://stash.run/docs/v2021.11.24/setup/install/community/#get-a-license).
+
+TODO: Note if you want to use an enterprise edition please folwo this link instead...
```

Comment:
Hello @scraly, shouldn't this line be removed ? Also "folwo" needs to be changed into "follow"

Reply:
sorry, I fixed it :)